modules/server/src/main/scala/seqexec/server/keywords/package.scala

```diff
@@ -133,6 +138,15 @@ package keywords {
       extends Keyword[Boolean](name, TypeBoolean, value)
   final case class StringKeyword(name: KeywordName, value: String)
       extends Keyword[String](name, TypeString, value)
+  final case class FloatPrecisionKeyword(name: KeywordName, precision: Int, value: Float)
+      extends Keyword[Float](name, TypeFloat, value)   {
+    override val stringValue: String =
+      s"%.${precision}f".formatLocal(new Locale("en", "US"), value.toDouble)
```

Comment:
I think we are launching with a set `Locale` to en_US, perhaps we can use the default?

Reply:
At any rate you may want to put the enUS locale in a separate variable. Instantiating a locale has a certain cost.

Reply:
Ok, I'll do.